### PR TITLE
It works

### PR DESCRIPTION
**Pull request by Cajam**
Completeness: 4/5

Comfort: 4/5

This could be a 5/5, but I am a little unsure on how to get it to work if I wrap all that code in a function or a variable, like in the bonus. Even so, I'm pretty happy I was able to get complete functionality with 18 lines of code. It almost makes me feel like I'm forgetting something.

### CODE REVIEW

File: timers.js [hunk 1]
```diff
@@ -1,4 +1,18 @@
-
-
-
-
+  var seconds = 0;
+  var resetTimer = document.getElementById("reset");
+    reset.addEventListener("click", function() {
+    timerID = document.getElementById('timer').textContent = "Stop Watch";
```
Comment (jshawl) [inline timers.js:4]: could also explicitly set `timerId` as a global outside this function.

File: timers.js [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+    seconds = 0;
+  });
+  var startTimer = document.getElementById("start");
+    start.addEventListener("click", function() {
+    startTimer = setInterval(updateTime, 1000);
```
Comment (jshawl) [inline timers.js:9]: try to use a different variable name to avoid confusion with line 7

File: timers.js [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+  });
+  var pauseTimer = document.getElementById("pause");
+    pause.addEventListener("click", function() {
```
Comment (jshawl) [inline timers.js:12]: indent left two spaces

**Comment by jshawl** [comment]
Holy smokes :+1: :star: fantastic solution! Would really like to see you tackle the bonus. Excellent job on this assignment.

**Comment by Cajam** [comment]
Thanks, I was proud I got it down to only about 18 lines of code. I looked
at other examples that were 50-100 lines. And Robin helped me figure out
how to solve the bonus as well yesterday afternoon. Thanks Jesse!

Very respectfully,

_Cassidy J. Harpster_
cassidy.j.harpster@gmail.com
(850) 218-4673
Washington, D.C.

**Pull request closed without merging**